bot/src/utilities/updateFeeds.js

```diff
@@ -0,0 +1,35 @@
+const { getFeed } = require("./getFeed");
+const { sendRssItems } = require("./sendRssItems");
+
+const updateFeeds = async (feedList, client) => {
+  const rssUrls = extractRssUrls(feedList);
+  const fetchedFeeds = await fetchFeeds(rssUrls);
+  for (const [channelId, rssUrl] of Object.entries(feedList)) {
+    const sendItems = await sendRssItems(
+      fetchedFeeds[rssUrl],
+      channelId,
+      client
+    );
+  }
+};
+
+const extractRssUrls = feedList => {
+  const rssUrlsSet = new Set();
+  for (const [channelId, rssUrl] of Object.entries(feedList)) {
```

Comment:
If you aren't using the `channelId` in this iterator, you can just do `Object.values()` instead of `Object.entries()`.

In this case, you also don't need to use an imperative loop. The `Set()` constructor takes an iterable as its argument, so you can just do `new Set(Object.values(feedList))` to get a unique set of feed URLs.